src/main/java/gui/pages/TraderListPage.java

```diff
@@ -0,0 +1,56 @@
+package gui.pages;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import gui.ReportBasePage;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+
+
+public class TraderListPage extends ReportBasePage {
+
+    private final SelenideElement traderListMenuItem = $("body > div:nth-child(2) > div:nth-child(1) > div:nth-child(3) > div:nth-child(1) > ul:nth-child(1) > li:nth-child(5) > ul:nth-child(2) > li:nth-child(3) > a:nth-child(1)");
+    private final SelenideElement reportsTraderListTimeframe = $(".form-control.period-picker-filter__display.au-target");
+    private final SelenideElement reportsTraderListSchema = $(By.id("schemaSelect"));
+
+    public TraderListPage traderListSelect(){
+        traderListMenuItem.click();
+       /* try {
+            // Pause the browser for 4 seconds
+            Thread.sleep(15000); // 4 seconds = 4000 milliseconds
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }*/
+        return this;
+    }
+    public TraderListPage reportsTraderListTimeframeSelect(String timeFrame){
+        reportsTraderListTimeframe.selectOption(timeFrame);
+        return this;
+    }
+
+    public TraderListPage reportsTraderListSchemaSortDesc(String schema){
```

Comment:
Keep spaces between brackets { and methods

Reply:
done